build/Containerfile.controlplane

```diff
@@ -1,16 +1,29 @@
 FROM --platform=$BUILDPLATFORM tonistiigi/xx AS xx-tools
 
-FROM --platform=$BUILDPLATFORM rust:alpine
-ARG TARGETPLATFORM
-ARG PROJECT_DIR=/workspace
-ARG BUILD_DIR=$PROJECT_DIR/build
+FROM --platform=$BUILDPLATFORM rust:alpine AS builder
 
 RUN apk add --no-cache clang lld
-COPY --from=xx-tools / /
 
 WORKDIR /workspace
+
+COPY --from=xx-tools / /
+
+ARG TARGETPLATFORM
+ARG PROJECT_DIR=/workspace
+ARG BUILD_DIR=$PROJECT_DIR/build
+
 RUN --mount=type=bind,source=../controlplane/src/,target=src \
     --mount=type=bind,source=../controlplane/Cargo.toml,target=Cargo.toml \
     --mount=type=bind,source=../controlplane/Cargo.lock,target=Cargo.lock \
     xx-cargo build --release --target-dir $BUILD_DIR && \
     xx-verify ./build/$(xx-cargo --print-target-triple)/release/controller
+
+RUN cp ./build/$(xx-cargo --print-target-triple)/release/controller /workspace/manager
```

Comment:
```suggestion
    xx-verify ./build/$(xx-cargo --print-target-triple)/release/manager

RUN cp ./build/$(xx-cargo --print-target-triple)/release/manager /workspace/manager
```

Reply:
After modified the `[[bin]].name` to `manager` in Cargo.toml, we can set the name of the binary file to `manager` directly after compilation.

Reply:
we don't want to change the `[[bin]].name` to `manager`. `manager` is a kuberentes convention for operators.

Reply:
https://github.com/kubernetes-sigs/blixt/blob/a322c2eec72e3323e09b5106b45312cf958142bb/config/manager/manager.yaml#L38
We should change the [[bin]].name to `manager` because we have used `./manger` to start the controller.
And by the way, we have added `cp ./build/{target}/release/controller /workspace/manager` in this PR, so IMO we can set the name of binary file to `manager` directly.

Reply:
sorry for the late reply,been travelling to attend Kubecon India 😅,and since we have used ./manger to start the controller,not sure why we dont need this line

Reply:
nit: I stll reconmond change binary name to manger after `cargo build` instead of `controller`. Otherwise lgtm.